json_tricks/encoders.py

```diff
@@ -278,6 +278,19 @@ def pathlib_encode(obj, primitives=False):
 
 	return {'__pathlib__': str(obj)}
 
+def slice_encode(obj, primitives=False):
+	if not isinstance(obj, slice):
+		return obj
+
+	if primitives:
+		return [obj.start, obj.stop, obj.step]
+	else:
+		return hashodict((
+			('__slice__', True),
```

Comment:
I slightly prefer nesting the data inside `__slice__`, but I've also used this pattern before with `__fraction__` , so let's just keep it.